src/com/google/enterprise/adaptor/DocumentHandler.java

```diff
@@ -185,6 +209,15 @@ private void initFullAccess(String gsaHostname, String[] fullAccessHosts) {
             + "access to content: {0}", new Object[] {addressesAndRanges});
   }
 
+  private boolean requestIsFromSkipCertAddresses(HttpExchange ex) {
```

Comment:
logic of server.skipCertCheckHosts will be added in next pull request. This merge request adds "server.skipCertCheckHosts" parameter to library, and passes it to DocumentHandler. I've decided to add requestIsFromSkipCertAddresses and addressIsInSkipCertAddresses, since they are simple and will be used later.